src/components/forms/driver/DriverForm.tsx

```diff
@@ -52,56 +52,62 @@ function DriverForm({
                   <RiCloseFill onClick={onCancel} />
                 </IconContext.Provider>
               </div>
-              <div className="flex flex-col gap-2">
-                <p className="text-xs text-transit-black-secondary font-medium required-field">Transporter</p>
-                <Controller
-                  rules={{ required: true }}
-                  control={control}
-                  render={({ field }) => (
-                    <TransporterPicker
-                      field={field}
-                    />
-                  )}
-                  name="transporter"
-                />
-                {errors.transporter && <ValidationError value="This field is required" />}
-              </div>
-              <div className="flex flex-col gap-2">
-                <p className="text-xs text-transit-black-secondary font-medium required-field">Driver Name</p>
-                <Input
+              <div className="flex flex-row gap-2">
+                <div className="flex flex-col gap-2" style={{ width: '50%' }}>
+                  <p className="text-xs text-transit-black-secondary font-medium required-field">Transporter</p>
+                  <Controller
+                    rules={{ required: true }}
+                    control={control}
+                    render={({ field }) => (
+                      <TransporterPicker
+                        field={field}
+                        isInvalid={Boolean(errors.transporter)}
+                      />
+                    )}
+                    name="transporter"
+                  />
+                  {errors.transporter && <ValidationError value="This field is required" />}
+                </div>
+                <div className="flex flex-col gap-2" style={{ width: '50%' }}>
```

Comment:
Did you try add `w-1/2` class to the `className="flex flex-col gap-2"`instead of `style={{ width: '50%' }}`? It should work.

Reply:
Yes. I tried and didn't work.